Formula/glib.rb

```diff
@@ -63,9 +63,13 @@ def install
     # `pkg-config --libs glib-2.0` includes -lintl, and gettext itself does not
     # have a pkgconfig file, so we add gettext lib and include paths here.
     gettext = Formula["gettext"].opt_prefix
+    lintl = " -lintl"
+    on_linux do
+      lintl = ""
+    end
     inreplace lib+"pkgconfig/glib-2.0.pc" do |s|
-      s.gsub! "Libs: -L${libdir} -lglib-2.0 -lintl",
-              "Libs: -L${libdir} -lglib-2.0 -L#{gettext}/lib -lintl"
+      s.gsub! "Libs: -L${libdir} -lglib-2.0#{lintl}",
+              "Libs: -L${libdir} -lglib-2.0 -L#{gettext}/lib#{lintl}"
```

Comment:
Do you still need `gettext` if you're not using `libintl`? It seems like you might not, since you're not passing a `-l` flag here.

Reply:
Right, I made the whole block Mac-only as it's not needed on Linux (and adding -lint flags on Linux breaks a lot of stuff everywhere, as lintl is provided by system glibc).